release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -273,13 +273,22 @@ If you are upgrading your cluster to {product-title} 4.6, old tokens from {produ
 [id="ocp-4-6-machine-api"]
 === Machine API
 
+[id="ocp-4-6-machine-api-multiple-block-device-mappings"]
+==== Support for multiple block device mappings
+
+The Machine API now supports multiple block device mappings. If more than one block device is given, you can now store logs, data in empty directory pods, and docker images in block devices that are separate from the root device on a machine.
+
+[id="ocp-4-6-default-validation-providerspec-apis"]
+==== Defaults and validation for the Machine API `providerSpec`
+
+Defaults and validation are now enabled on a particular cloud provider API before input from the `providerSpec` is persisted to etcd. Validation is run against machines and MachineSets when they are created. Feedback is returned when the configuration is known to prevent machines from being created by the cloud provider. For example, a MachineSet is rejected if location information is required but is not provided. The Machine API Operator (MAO) exposes both a `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` to the Cluster Version Operator (CVO), which run over the machine resource so that each provider can define their API defaults and validation.
```

Comment:
> The Machine API Operator (MAO) exposes both a `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` to the Cluster Version Operator (CVO), which run over the machine resource so that each provider can define their API defaults and validation.

I would be tempted to drop this sentence, it's an implementation detail and I don't think it adds much to the understanding for users